### PR TITLE
add .aderc file and update instruction to setup ade environment

### PR DESCRIPTION
**Pull request by mitsudome-r**
This adds .aderc file to setup ADE envorinment for ROS2 foxy.

To test please run the instructions to setup ADE in the updated doc and confirm that colcon build works in ADE.

Here's the list of commands copied from the updated doc:
```
$ mkdir adehome
$ cd adehome
$ touch .adehome
$ git clone https://github.com/tier4/AutowareArchitectureProposal
$ cd AutowareArchitectureProposal
$ git checkout ros2
$ mkdir src
$ vcs import src < autoware.proj.repos
$ ade start --enter
$ cd AutowareArchitectureProposal
$ colcon build
```

### CODE REVIEW

**Comment by mitsudome-r** [comment]
@nnmm Could you test this PR?

File: docs/porting-to-ROS2.md [hunk 1]
```diff
@@ -2,23 +2,28 @@ Porting ROS1 code to ROS2
 =======================
 
 ## Setting up the environment
-Make sure you have an empty `.adehome` file in `~/ade-home`, then clone [`AutowareArchitectureProposal`](https://github.com/tier4/AutowareArchitectureProposal) into that directory.
+Do following to setup and start ADE environment
```
Comment (esteve) [inline docs/porting-to-ROS2.md]: s/Do following/Do the following/

**Review by esteve — APPROVED** [review]

**Comment by esteve** [comment]
@mitsudome-r there's one small typo, but looks good overall.

File: docs/porting-to-ROS2.md [hunk 2]
```diff
@@ -2,23 +2,28 @@ Porting ROS1 code to ROS2
...
+```
+$ mkdir ade-home
+$ cd ade-home
+$ touch .adehome
+```
 
-Then start it:
+2. Setup AutowareArchitectureProposal
+```
+$ git clone https://github.com/tier4/AutowareArchitectureProposal
```
Comment (nnmm) [inline docs/porting-to-ROS2.md:17]: adehome -> ade-home

File: docs/porting-to-ROS2.md [hunk 3]
```diff
@@ -2,23 +2,28 @@ Porting ROS1 code to ROS2
...
+$ cd AutowareArchitectureProposal
+$ git checkout ros2
+```
 
-    cd ~/ade-home/AutowareArchitectureProposal
-    git checkout ros2
-    ade start --enter
+3. enter ADE
+```
+$ cd ~/adehome/AutowareArchitectureProposal
```
Comment (nnmm) [inline docs/porting-to-ROS2.md]: adehome -> ade-home

File: docs/porting-to-ROS2.md [hunk 4]
```diff
@@ -2,23 +2,28 @@ Porting ROS1 code to ROS2
...
-    export ADE_DOCKER_RUN_ARGS="--cap-add=SYS_PTRACE -e RMW_IMPLEMENTATION=rmw_cyclonedds_cpp"
-    export ADE_GITLAB=gitlab.com
-    export ADE_REGISTRY=registry.gitlab.com
-    export ADE_DISABLE_NVIDIA_DOCKER=true
-    export ADE_IMAGES="
-      registry.gitlab.com/autowarefoundation/autoware.auto/autowareauto/amd64/ade-foxy:master
-    "
+1. Setup ade home
+```
+$ mkdir ade-home
```
Comment (nnmm) [inline docs/porting-to-ROS2.md:10]: Maybe add a `cd ~` before, since the following commands assume this

**Review by nnmm — APPROVED** [review]
LGTM with a few small comments

**Comment by mitsudome-r** [comment]
@nnmm @esteve Thank you for the review. I fix the instruction as suggested.

**Pull request merged**